perllib/FixMyStreet/Cobrand/Bexley/Garden.pm

```diff
@@ -50,26 +52,61 @@ sub garden_current_subscription {
     my $contract = $customer->{ServiceContracts}[0];
     return unless $contract;
 
-    my $parser
-        = DateTime::Format::Strptime->new( pattern => '%d/%m/%Y %H:%M' );
-    my $end_date = $parser->parse_datetime( $contract->{EndDate} );
+    # XXX should maybe sort by CreatedDate rather than assuming first is OK
+    $sub->{cost} = try {
+        my ($payment) = grep { $_->{PaymentStatus} eq 'Paid' } @{ $contract->{Payments} };
+        return $payment->{Amount};
+    };
+
+    my $parser = DateTime::Format::Strptime->new( pattern => '%d/%m/%Y %H:%M' );
+    $sub->{end_date} = $parser->parse_datetime( $contract->{EndDate} );
+
+    $sub->{customer_external_ref} = $customer->{CustomerExternalReference};
+
+    $sub->{bins_count} = $contract->{WasteContainerQuantity};
+
+    my $c = $self->{c};
+    my $p = $c->model('DB::Problem')->search({
```

Comment:
I think we may need to change `get_original_sub` in Controller/Waste as well (or special-case it so it calls this subroutine for Bexley, to get the 'row' bit).
The current Waste code for other cobrands makes use of both `get_original_sub` and `garden_current_subscription`, looks like the former is for DB lookup and the latter for getting the garden service from the list of services...? I don't know if we should be keeping this distinction here.